### ISSUE TITLE
WCAG2 contrast errors on search result paging

### ISSUE THREAD

**Issue by mberkowski** [UMNLibraries/cdm-blacklightify#159]
Low contrast on Blacklight default CSS for search result paging. Eventually this may get revised with site redesigns, but for now just darken these to make contrast errors go away

<img width="544" height="221" alt="Image" src="https://github.com/user-attachments/assets/1caee771-e0fc-4f6e-933e-146c028d231d" />

**Comment by mberkowski** [comment]
Also "Back to search"

<img width="373" height="169" alt="Image" src="https://github.com/user-attachments/assets/d1c6353a-f5aa-4c33-864d-f2153e223d13" />

**Comment by tantz001** [comment]
color now matches  page info in search header.
